validator/client/beacon-api/beacon_api_validator_client.go

```diff
@@ -144,8 +144,8 @@ func (*beaconApiValidatorClient) WaitForActivation(_ context.Context, _ *ethpb.V
 }
 
 // Deprecated: Do not use.
```

Comment:
Do you need to remove this?

Reply:
I added this comment because `WaitForChainStart` is technically deprecated (according to validator.pb.go), but it's still used throughout the codebase.